src/System.Runtime.WindowsRuntime/src/System/IO/UnmanagedMemoryStreamInternal.cs

```diff
@@ -203,7 +203,7 @@ public override long Position
                 {
                     // On 32 bit machines, ensure we don't wrap around.
                     if (value > (long)Int32.MaxValue || _mem + value < _mem)
-                        throw new ArgumentOutOfRangeException("value", SR.ArgumentOutOfRange_StreamLength);
+                        throw new ArgumentOutOfRangeException(nameof(Position), SR.ArgumentOutOfRange_StreamLength);
```

Comment:
I misspoke in the previous set of comments where @jamesqo had recommended keeping value.  We should keep value here.  Please revert this.